akka-docs/src/main/paradox/index-cluster.md

```diff
@@ -7,7 +7,10 @@ For the new API see @ref[Cluster](typed/index-cluster.md).
 
 @@@ index
 
+* [cluster-specification](common/cluster.md)
 * [cluster-usage](cluster-usage.md)
+* [cluster](common/cluster-membership.md)
+* [cluster](common/cluster-background.md)        
```

Comment:
The ToC for Classic Clustering and Cluster are somewhat inconsistent now. Maybe those should be added to the (Typed) Cluster ToC too?